Units/ocaml_empty.ml_xref.t/expected.tags-x

```diff
@@ -1 +0,0 @@
-Input            module        1 input.ml 
```

Comment:
Just a question; ECB refers tags file, TAGS file or -x output?
(I don't want to change tags output.)